selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestProfileServiceClient.java

```diff
@@ -1,55 +0,0 @@
-/*
```

Comment:
This class is being used in the [EditorSettingsTest](https://github.com/codenvy/codenvy/blob/5.18.0/selenium/codenvy-selenium-test/src/test/java/com/codenvy/selenium/preferences/EditorSettingsTest.java#L83-L83) and in the [OrganizationTest](https://github.com/codenvy/codenvy/blob/5.18.0/selenium/codenvy-selenium-test/src/test/java/com/codenvy/selenium/dashboard/organization/OrganizationTest.java#L72-L72). So, removal of it will definitely break Codenvy onprem tests.